src/main/java/com/mraof/minestuck/entity/LotusFlowerEntity.java

```diff
@@ -52,11 +52,14 @@ public class LotusFlowerEntity extends LivingEntity implements IAnimatable, IEnt
 	
 	// Animation lengths
 	private static final int OPENING_LENGTH = 120;        //6 sec open animation * 20 ticks/sec = 120
-	private static final int OPEN_IDLE_LENGTH = 320;    //4 sec idle animation * 4 loops * 20 ticks/sec = 320
+	private static final int GROWTH_LENGTH = 21;		//1.04 sec grow animation * 20 ticks/sec = ~21
+	private static final int OPEN_IDLE_LENGTH = 240;    //4 sec idle animation * 3 loops * 20 ticks/sec = 240
 	private static final int VANISHING_LENGTH = 13;        //0.65 sec vanish animation * 20 ticks/sec = 13
 	
 	// Animation start times
 	private static final int IDLE_TIME = -1;
+	private static final int GROWTH_STOP = IDLE_TIME - 1;
+	private static final int GROWTH_START = GROWTH_STOP - GROWTH_LENGTH;
```

Comment:
Start and end times going into the negatives seem a bit odd. Perhaps add some documentation that gives an overview on these start times, or rework the animation times into something more intuitive.